edifact-edi/2.3/modules/ROOT/pages/edifact-edi-connector-batching.adoc

```diff
@@ -0,0 +1,60 @@
+= EDIFACT EDI Connector 2.3 - Batch Outbound EDIFACT Messages - Mule 4
+
+For EDI messages that are not time-sensitive, such as invoice and payment remittances, you can use the Batch Write operation to merge multiple individual messages into a single interchange to send to your trading partners.
```

Comment:
Something to think about: where it is possible, _retrain_ yourself (I am trying for myself!) to recast modal uses like "you can use" to just "use," per our style ref. and https://confluence.internal.salesforce.com/display/public/DOCTEAM/modal+verbs.

Reply:
```suggestion
For EDI messages that are not time-sensitive, such as invoice and payment remittances, use the Batch Write operation to merge multiple individual messages into a single interchange to send to your trading partners.
```